cmd/kubectl-gadget/utils/trace.go

```diff
@@ -460,7 +460,7 @@ func PrintTraceOutputFromStream(traceID string, expectedState string, params *Co
 // PrintTraceOutputFromStatus is used to print trace output using function
 // pointer provided by caller.
 // It will parse trace.Spec.Output and print it calling the function pointer.
-func PrintTraceOutputFromStatus(traceID string, expectedState string, customResultsDisplay func(results *gadgetv1alpha1.TraceList)) error {
+func PrintTraceOutputFromStatus(traceID string, expectedState string, customResultsDisplay func(results *gadgetv1alpha1.TraceList) error) error {
```

Comment:
Good point! I was doubting if I needed to do this when I first implement the API.
Nonetheless, I would say in this case the code should become:
```golang
// Old version:
customResultsDisplay(&traces)

return nil

// New version:
return customResultsDisplay(&traces)
```

Reply:
Oops, for some git issue this change got lost 😅 . Thanks for noticing it!